nautobot/docs/additional-features/template-filters.md

```diff
@@ -149,15 +147,15 @@ Return x/y as a percentage.
 
 ### placeholder
 
-Render a muted placeholder (`<span class="text-muted">&mdash;</span>`) if value is falsey, else render the provided value.
```

Comment:
Falsey is a thing, meaning false, none, 0, etc

Reply:
TIL.  Reverted